src/types/__tests__/effector/effect.test.js

```diff
@@ -355,9 +355,6 @@ describe('#filterMap', () => {
       "
       --typescript--
       Type 'Event<number>' is not assignable to type 'Event<number | void>'.
```

Comment:
```js
Type 'Event<number>' is not assignable to type 'Event<number | void>'
```

It is important to put "this" overload after the first one to keep this message right.